src/set_theory/cardinal_ordinal.lean

```diff
@@ -229,6 +230,47 @@ by rw [← aleph_zero, ← aleph_succ, ordinal.succ_zero]
 lemma countable_iff_lt_aleph_one {α : Type*} (s : set α) : countable s ↔ #s < aleph 1 :=
 by rw [← succ_omega, lt_succ, mk_set_le_omega]
 
+/-- Ordinals that are cardinals are unbounded. -/
+theorem ord_aleph'_unbounded : unbounded (<) {b : ordinal | b.card.ord = b} :=
+unbounded_lt_iff.2 $ λ a, ⟨_, ⟨(by { dsimp, rw card_ord }), (lt_ord_succ_card a).le⟩⟩
+
+theorem eq_aleph'_of_eq_card_ord {o : ordinal} (ho : o.card.ord = o) : ∃ a, (aleph' a).ord = o :=
+⟨cardinal.aleph_idx.rel_iso o.card, by { simp, exact ho }⟩
```

Comment:
This simp looks nonterminal? Does `simp [ho]` or `simpa using ho` work instead?

Reply:
`simpa using ho` does the job.